tools/shell/embedded_shell.cpp

```diff
@@ -11,10 +11,12 @@
 #include <sstream>
 
 #include "catalog/catalog.h"
+#include "catalog/catalog_entry/node_table_catalog_entry.h"
+#include "catalog/catalog_entry/rel_table_catalog_entry.h"
```

Comment:
I think we don't need these two includes.

Reply:
I think we do, getNodeTables/getRelTables returns a vector<NodeTableCatalogEntry*> and vector<RelTableCatalogEntry*>. The shell needs to know the types.